addons/website_sale/models/product_template.py

```diff
@@ -759,7 +759,7 @@ def _search_get_detail(self, website, order, options):
         if tags:
             if isinstance(tags, str):
                 tags = tags.split(',')
-            domains.append([('product_variant_ids.all_product_tag_ids', 'in', tags)])
+            domains.append([('product_variant_ids.all_product_tag_ids.id', 'in', tags)])
```

Comment:
Thank for the fix :duck: 

Why in saas-18.1?
I see this domain was added in 17.0 https://github.com/odoo/odoo/pull/124299 and not changed since then

Can you add exact details what is causing this issue in saas-18.1?